src/pages/Home/Home.jsx

```diff
@@ -0,0 +1,156 @@
+import React from "react";
+import { NavLink, Outlet } from "react-router-dom";
+import { logo } from "../../assets";
+import "./home.css";
+
+function Home() {
+  return (
+    <div className="bg-[#11161b] text-gray-50 h-screen w-screen flex flex-col">
+      <nav className="navbar bg-base-100 px-4 py-2">
+        <div className="flex-none">
+          <img src={logo} alt="logo" className="h-10" />
+        </div>
+        <div className="flex-1">
+          <button className="ff-nickainley text-2xl mx-4">The Social</button>
+        </div>
+        <div className="dropdown dropdown-end">
+          <label tabIndex="0" className="btn m-1 border-0">
+            <div className="indicator text-2xl">
+              <span className="indicator-item badge rounded-full ">4</span>
+              <div className="grid w-10 h-10 bg-base-300 place-items-center">
+                <i className="fa-solid fa-bell"></i>
+              </div>
+            </div>
+          </label>
+          <ul
+            tabIndex="0"
+            className="dropdown-content menu p-2 shadow bg-base-100 rounded-box w-52">
+            <li className="bg-[#454d50] hover:bg-[#2c3133]">
+              <p>Notification 1</p>
```

Comment:
could've put text directly inside li tag

Reply:
ok will update that
thank you